src/target/move.c

```diff
@@ -361,22 +376,6 @@ static void move_side(enum step_direction side)
 	entered_next_cell();
 }
```

Comment:
I would disable_walls_control at the end of the function

Reply:
I am not 100% sure that it would not have any side effects, so I rather not change that.

That code is the one used during exploration and I would prefer to avoid much changes there before OSHWDem. :wink:

Reply:
Ok, you are right, I thought it was new